libraries/core/src/test/kotlin/com/bumble/appyx/core/store/RetainedInstanceStoreTest.kt

```diff
@@ -46,6 +46,44 @@ class RetainedInstanceStoreTest {
         assertEquals(2L, longValue)
     }
 
+    @Test
+    fun `GIVEN object stored with int type WHEN second object stored with int type THEN second factory not called`() {
```

Comment:
I don't think we should test it. We're not testing `RetainedInstanceStore` contract here but internal implementation. 
[This test](https://github.com/bumble-tech/appyx/pull/361/commits/5d629d2a2dfdc180c6363c920b9177762ca1151e#diff-40f24b8c2f8f8ef099fcc890959e97345f158cad3c9a27dfa8eb03d1f06140caR76) covers this particular scenario

Reply:
fair enough. removed